onnxruntime/test/onnx/TestCase.cc

```diff
@@ -430,25 +445,23 @@ Status OnnxTestCase::ParseConfig() {
        to represent simple config files that have only key-value pairs.
      */
     std::map<std::string, std::string> fc;
+    per_sample_tolerance_ = 1e-3;
+    relative_per_sample_tolerance_ = 1e-5;
+#ifdef USE_CUDA
+    relative_per_sample_tolerance_ = 0.017;  // to resolve random MNIST test failure
+#endif
+    post_processing_ = false;
     if (read_config_file(config_path, fc)) {
-      if (fc.count("per_sample_tolerance")) {
+      if (fc.count("per_sample_tolerance") > 0) {
```

Comment:
```! fc.empty()``` is more readable unless you want to check the count against a different number in the future.

Reply:
The code
```
if(fc.count("per_sample_tolerance"))
```
will generates a warnings, because fc.count returns an int, but 'if' expects a bool.

I don't understand how "! fc.empty()" could help.